src/library_webaudio.js

```diff
@@ -252,11 +252,11 @@ let LibraryWebAudio = {
       // '_wpn' == 'Worklet Processor Name', use a deliberately mangled name so
       // that this field won't accidentally be mixed with user submitted
       // messages.
-      _wpn: UTF8ToString(HEAPU32[options]),
-      audioParams,
-      contextHandle,
-      callback,
-      userData
+      '_wpn': UTF8ToString(HEAPU32[options]),
+      'audioParams': audioParams,
+      'contextHandle': contextHandle,
+      'callback': callback,
+      'userData': userData
```

Comment:
Is there some way we can annotate this object such that closure won't mangle these fields?

Reply:
I looked into using `/** dict */` but that didn't seem to work.  Sadly, I can't see any other way to try.

Reply:
I was looking into the annotations but didn’t see anything either that would carry the naming across files.

Reply:
Can we add a comment here at least.  Something like:

```
Sadly we cannot use the shorthand object notiation here since closure would then minify the field names.
```

Reply:
I shortened the names and commented, which allows the generated minified code to look like:
```
_wpn: X(G[b]),
ap: f,
ch: a,
cb: e,
ud: c
```